### PR TITLE
9 cleanup code to make it more consistent

### PR DESCRIPTION
**Pull request by jbowkett**
First part of this pull request - mostly code formatting and moving code around.  I thought this would be a good place for someone to check and merge, then I'll raise a further PR tomorrow with some of the more material changes, i.e. separating out the messaging etc. and the config

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ install:
   - nohup bash -c "cd kafka && bin/zookeeper-server-start.sh config/zookeeper.properties &"
   - nohup bash -c "cd kafka && bin/kafka-server-start.sh config/server.properties &"
   - sleep 5
-  - kafka/bin/kafka-topics.sh --create --partitions 1 --replication-factor 1 --topic org.mache.integrations.eventing.TestEventingBase.TestOtherEntity --zookeeper localhost:2181
+  - kafka/bin/kafka-topics.sh --create --partitions 1 --replication-factor 1 --topic TestEventingBase.TestOtherEntity --zookeeper localhost:2181
```
Comment (alex-smith-excelian) [inline .travis.yml:23]: Will we lose anything from the package renaming?

File: JMeter-Projects/JMeter Mongo Test1.jmx [hunk 2]
```diff
@@ -75,7 +75,7 @@
                 </elementProp>
               </collectionProp>
             </elementProp>
-            <stringProp name="classname">org.mache.jmeter.mongo.MongoRandomLoad</stringProp>
+            <stringProp name="classname">MongoRandomLoad</stringProp>
```
Comment (alex-smith-excelian) [inline JMeter-Projects/JMeter Mongo Test1.jmx:78]: Won't we need the full package name for the class?

File: JMeter-Projects/Read & Write Through Performance - Cassandra.jmx [hunk 3]
```diff
@@ -148,7 +148,7 @@
                 </elementProp>
               </collectionProp>
             </elementProp>
-            <stringProp name="classname">org.mache.jmeter.cassandra.WriteToCassandra</stringProp>
+            <stringProp name="classname">WriteToCassandra</stringProp>
```
Comment (alex-smith-excelian) [inline JMeter-Projects/Read & Write Through Performance - Cassandra.jmx:151]: Same as above

File: couchbase-ns/src/test/java/CouchbaseCacheLoaderIntegrationTest.java [hunk 4]
```diff
@@ -30,11 +30,11 @@
...
 
-    private CacheThing<String, TestEntity> cache;
+    private MacheImpl<String, TestEntity> cache;
 
     @Before
     public void setup() {
-        cache = new CacheThing<>(new CouchbaseCacheLoader<>(CouchbaseConfig.builder()
+        cache = new MacheImpl<>(new CouchbaseCacheLoader<>(CouchbaseConfig.builder()
                 .withServerAdresses(Collections.singletonList(EXCELIAN_COUCHBASE))
                         //.withServerAdresses(Collections.singletonList(LOCAL_COUCHBASE))
```
Comment (alex-smith-excelian) [inline couchbase-ns/src/test/java/CouchbaseCacheLoaderIntegrationTest.java:39]: Can we remove the commented lines?

File: couchbase-ns/src/main/java/com/excelian/mache/couchbase/CouchbaseConfig.java [hunk 5]
```diff
@@ -1,170 +1,170 @@
...
-
-    public SchemaOptions getSchemaOptions() {
-        return schemaOptions;
-    }
-
-    public CouchbaseEnvironment getCouchbaseEnvironment() {
-        return couchbaseEnvironment;
-    }
-}
+package com.excelian.mache.couchbase;
```
Comment (alex-smith-excelian) [inline couchbase-ns/src/main/java/com/excelian/mache/couchbase/CouchbaseConfig.java:1]: We appear to be using the lambda builder pattern for the Cassandra and Mongo builder classes, is this something we will want to use here?

**Comment by jbowkett** [comment]
@alex-smith-excelian has reviewed and approved

**Pull request merged**